harmony/harmonytask/task_type_handler.go

```diff
@@ -346,10 +346,12 @@ func (h *taskTypeHandler) AssertMachineHasCapacity() error {
 		return xerrors.Errorf("Did not accept %s task: out of available GPU: required %f available %f)", h.Name, h.Cost.Gpu, r.Gpu)
 	}
 
+	/*
```

Comment:
I don't know what's going on here but I've had this cause problems on both Curio nodes I've tried to run; I thought it was because it insisted on having a sector size's space available but on the first server where I had this be a problem on it was set to 64G sectors and I freed up over 90G and it still complained about this. On the server I'm now running mkv2 on I have 402G free but every "piece" task fails with this so I give up.